### PR TITLE
Allow temperature ramp

### LINKED ISSUE #5: Create option to run an annealed simulation
Right now, the default behaviour is to run at one kT, so if you wanted to set up a quenched simulation you would have to utilize some sort of restart logic where a new simulation is generated from the ending state point of the last simulation.  

It wouldn't be too hard to be able to make this choice before starting any simulations, and build in the temperature steps into a single simulation.

A couple of considerations: 

1. This might require reconfiguration of the Simulation class in `sim.py`.  Right now it has a single `run` method, which is essentially a quenched run.  So, one solution might be to rename that to `run_quench` then add a `run_anneal` method as well

2. This would require some additional parameters in planckton-flow, and setting up the workspace and simulation instructions.

Here is a simple example of how it might work:

```
# schedule is a dictionary of temperature : n_steps
        last_time_step = shrink_time
        for kT in schedule:
            n_steps = schedule[kT]
            integrator.set_params(kT=kT)
            integrator.randomize_velocities(seed=42)
            hoomd.run(last_time_step + n_steps)
            last_time_step += n_steps
```

### PR DESCRIPTION
**Pull request by jennyfothergill**
Fixes #5

### CODE REVIEW

**Comment by erjank** [comment]
nice


Eric Jankowski PhD | he/him/his
Associate Professor | Micron School of Materials Science & Engineering
Boise State University | 1910 W University Dr. | Boise, ID 83725-2090
***@***.*** | (208) 426-5681
Zoom: https://boisestate.zoom.us/j/3783842550
Book my calendar: https://ericjankowski.youcanbook.me/

File: planckton/sim.py [hunk 1]
```diff
@@ -229,23 +241,26 @@ def run(self):
...
 
-            # After shrinking, reset velocities and change temp
-            integrator.set_params(kT=self.kT)
-            integrator.randomize_velocities(seed=42)
-            integrator_mode.set_params(dt=self.dt)
+            # Begin temp ramp
+            for kT, tau, n_steps in zip(self.kT, self.tau, self.n_steps):
+                integrator.set_params(kT=kT, tau=tau)
+                # Reset velocities
+                integrator.randomize_velocities(seed=42)
```
Comment (chrisjonesBSU) [inline planckton/sim.py:253]: Wouldn't you want `seed` to be a parameter that can be changed? Would that be useful for running repeat simulations if needed?

**Comment by chrisjonesBSU** [comment]
Looks good to me. I think handling list of `n_steps` would be simpler if you used `hoomd.run` rather than `hoomd.run_upto` but if you already have the logic written to use `run_upto` then you might as well keep it.

File: planckton/sim.py (hunk 1 above)
Comment (jennyfothergill) [reply planckton/sim.py:253]: Yeah, that would be a useful change. (For instance if we wanted to do replicate sims.)

**Pull request merged**